depthai_examples/launch/multi_stereo_inertial_node.launch

```diff
@@ -0,0 +1,220 @@
+
+    <arg name="cam_pos_x"             default="0.0" /> <!-- Position respect to base frame (i.e. "base_link) -->
+    <arg name="cam_pos_y"             default="0.0" /> <!-- Position respect to base frame (i.e. "base_link) -->
+    <arg name="cam_pos_z"             default="0.0" /> <!-- Position respect to base frame (i.e. "base_link) -->
+    <arg name="cam_roll"              default="0.0" /> <!-- Orientation respect to base frame (i.e. "base_link) -->
+    <arg name="cam_pitch"             default="0.0" /> <!-- Orientation respect to base frame (i.e. "base_link) -->
+    <arg name="cam_yaw"               default="0.0" /> <!-- Orientation respect to base frame (i.e. "base_link) -->
+
+    <arg name="lrcheck"               default="true" /> 
+    <arg name="extended"              default="false"/> 
+    <arg name="subpixel"              default="true" />
+    <arg name="rectify"               default="true" />
+    <arg name="depth_aligned"         default="false" />
+
+    <arg name="enableSpatialDetection" default="false" />
+    <arg name="syncNN"                 default="false" />
+    <arg name="detectionClassesCount"  default="80"   />
+    <arg name="nnName"                 default="x"         />
+    <arg name="resourceBaseFolder"     default="$(find depthai_examples)/resources" />
+    <arg name="nnPath"                 default="" /> 
+
+     <!-- see https://docs.luxonis.com/projects/api/en/latest/components/nodes/stereo_depth/#currently-configurable-blocks for possible combination of optionf for higher fps-->
+    <arg name="stereo_fps"            default="40"  />
+    <arg name="confidence"            default="90" />
+    <arg name="LRchecktresh"          default="1"   />
+    <arg name="monoResolution"        default="720p"/>
+    <arg name="angularVelCovariance"  default="0"   />
+    <arg name="linearAccelCovariance" default="0"   />
+
+    <arg name="enableDotProjector"    default="true"/>
+    <arg name="enableFloodLight"      default="true"/>
+    <arg name="dotProjectormA"        default="400.0"/>
+    <arg name="floodLightmA"          default="400.0"/>
+    <arg name="enableRviz"            default="false"/>
+    <arg name="enableMarkerPublish"   default="false" />
+
+    <include file="$(find depthai_bridge)/launch/urdf.launch">
+        <arg name="base_frame"      value="$(arg  base_frame)"  />
+        <arg name="parent_frame"    value="$(arg  parent_frame)"/>
+        <arg name="camera_model"    value="$(arg  camera_model)"/>
+        <arg name="tf_prefix"       value="$(arg  tf_prefix)" />
+        <arg name="cam_pos_x"       value="$(arg  cam_pos_x)"   />
+        <arg name="cam_pos_y"       value="$(arg  cam_pos_y)"   />
+        <arg name="cam_pos_z"       value="$(arg  cam_pos_z)"   />
+        <arg name="cam_roll"        value="$(arg  cam_roll)"    />
+        <arg name="cam_pitch"       value="$(arg  cam_pitch)"   />
+        <arg name="cam_yaw"         value="$(arg  cam_yaw)"     />
+    </include>
```

Comment:
you are launching only 1 urdf with TF. How are you publishing TF for the other 2 ?

Reply:
You are right, I left it open since I was not using URDF for my purposes. After second thought, I agree, we can exclude the launch file.

Reply:
We can exclude the launch file or make changes in the single device launch file. And call it multiple times in your launch file to launch multiple devices. 
I think that would look good.